packages/webapp/src/__mock-data__/test-application.js

```diff
@@ -2,6 +2,19 @@ const dataString = `
 {
   "registrationTaskDetails": {
     "taskList": [
+      {
+        "taskTitle": "Applicant information",
+        "tasks": [
+          {
+            "title": "Add details about the person applying",
+            "status": "COMPLETED",
+            "completedTaskUrl": "/land/applicant-details-is-agent",
+            "startTaskUrl": "/land/applicant-details-is-agent",
+            "inProgressUrl": "/land/add-landowners",
+            "id": "applicant-details-is-agent"
```

Comment:
This does not match the id in [packages/webapp/src/utils/register-task-list.js](https://github.com/DEFRA/biodiversity-net-gain-service/pull/482/files#diff-a52c8b94614c8e2ba7fdd4c53843fe7cdedf2d333160e04eba4aefe6f73f8bb1)